src/encoded/schemas/file.json

```diff
@@ -268,6 +270,7 @@
             "type": "string",
             "oneOf": [
```

Comment:
what is the difference between a oneOf and a regular enum?  The comment looks like it's in a weird spot.